install/kubernetes/cilium/charts/operator/templates/clusterrole.yaml

```diff
@@ -57,3 +57,26 @@ rules:
   - get
   - list
   - watch
+{{- if or (ge .Capabilities.KubeVersion.Minor "14") (gt .Capabilities.KubeVersion.Major "1") }}
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  # To be used during HA deployment of cilium-operator in order
+  # to obtain lease lock for leader election.
+  - leases
+  verbs:
+  - '*'
+{{- end }}
+{{- if and (lt .Capabilities.KubeVersion.Minor "14") (eq .Capabilities.KubeVersion.Major "1") }}
```

Comment:
I think it would be helpful to add a comment for why we are checking for that specific K8s version.

Reply:
Will do.